plugins/CoreAdminHome/Commands/DeleteConfig.php

```diff
@@ -0,0 +1,113 @@
+<?php declare(strict_types=1);
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\CoreAdminHome\Commands;
+
+use Piwik\Config;
+use Piwik\Plugin\ConsoleCommand;
+use Piwik\Plugins\CoreAdminHome\Commands\DeleteConfig\ConfigDeletingManipulation;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DeleteConfig extends ConsoleCommand
+{
+    protected function configure()
+    {
+        $this->setName('config:delete');
+        $this->setDescription('Delete one or more config settings in the file config/config.ini.php');
+        $this->addArgument('assignment', InputArgument::OPTIONAL | InputArgument::IS_ARRAY, "List of config setting assignments, eg, section, section.config_setting_key or section.config_setting_key[position]");
+        $this->addOption('section', null, InputOption::VALUE_REQUIRED, 'The section the INI config setting belongs to.');
+        $this->addOption('key', null, InputOption::VALUE_REQUIRED, 'The name of the INI config setting.');
+        $this->setHelp("This command can be used to remove INI config settings on a Piwik instance.
+
+You can remove config values two ways, via --section, --key or by command arguments.
+
+To use --section, --key, simply supply those options. You can remove one whole section or one key inside the section.
+
+To use arguments, supply one or more arguments in the following format:
+
+Remove section:
+$ ./console config:delete 'section'
+'section' is the name of the section,
+
+Remove setting inside section
+$ ./console config:delete 'section.config_setting_key'
+'section' is the name of the section,
+'config_setting_key' the name of the setting
+
+To remove an array setting position, supply an argument like this:
+$ ./console config:set 'section.config_setting_key[position]'
```

Comment:
@javier-de-juan in case you don't need this feature be great to remove it as well just as it adds a bit of complexity and people might rarely know the correct index of an array etc. Happy to keep it though if needed.

Reply:
The same with this, I could remove it. I'm going to wait for someone with different opinion and If there's no comments today I will change it and give just the posibility of remove settings in sections ;)

Reply:
Thanks. Be great to remove it and if needed it can be always added quickly. This way it's less things to maintain that otherwise maybe nobody would use etc and simplifies the documentation, the code etc. Let me know once you made the changes and we'll review 👍  Thanks for this, very appreciated.

Reply:
This feature was completely removed from the code ;)